src/main/assembly/src.xml

```diff
@@ -45,7 +45,7 @@
       <outputDirectory>lib</outputDirectory>
     </file>
     <file>
-      <source>${basedir}/bin/run-local-app.sh</source>
+      <source>${basedir}/bin/run-wikipedia-standalone-application.sh</source>
```

Comment:
Just curious: Did you mean to replace run-local-app.sh with run-wikipedia-standalone-application.sh in `assembly/src.xml_` ? Does run-local-app.sh not exist anymore?